tsp-output/@azure-tools/typespec-csharp/src/Generated/Models/ChatCompletionResponseMessage.Serialization.cs

```diff
@@ -10,9 +10,9 @@
 
 namespace OpenAI.Models
 {
-    public partial class ChatCompletionResponseMessage : IJsonModel<ChatCompletionResponseMessage>
+    public partial class ChatCompletionResponseMessage : JsonModel<ChatCompletionResponseMessage>
```

Comment:
Inherits from `JsonModel` abstract class in System.ClientModel to add `AdditionalProperties` dictionary.